webpack.config.babel.js

```diff
@@ -34,7 +35,10 @@ const WrapperPlugin = require('wrapper-webpack-plugin');
 
 
 const staticSiteGeneratorConfig = {
-  paths: Object.keys(ROUTES).map(key => ROUTES[key]),
+  paths: [
+    ...Object.keys(ROUTES).map(key => ROUTES[key]),
```

Comment:
Is this ordering conscious? What happens when someone tries to add a route that already exists in `REDIRECT_ROUTES` .. they'll get confused because it redirects to somewhere else. Perhaps this is a good thing - i.e. we wont break existing redirects.